pages/index.jsx

```diff
@@ -120,7 +121,7 @@ function Home() {
             <h2>Upcoming Events</h2>
             <hr />
             <Row className="justify-content-center">
-              {eventCards}
+              {eventCards ? eventCards : <Card><Card.Text>Loading...</Card.Text></Card>}
```

Comment:
Maybe should be `eventCards ?? <Card><Card.Text>Loading...</Card.Text></Card>`?

Reply:
(Or use `||` if you want to include falsey values for `eventCards`)

Reply:
Fixed in f8f64b1992724eb315c202799ff5ecab137011cd